docs/alternatives/index.md

```diff
@@ -28,9 +28,31 @@ to the mature and widely available C++14 standard. Key features include:
 - Intelligent, unit-aware functions for rounding and computing inverses.
 - Minimal friction by using a single, short namespace: everything's in `au::`.
 
+## Alternatives considered here
+
+We'll consider several of the most prominent alternatives in more detail.  While there are [many
+more libraries](https://github.com/topics/dimensional-analysis?l=c%2B%2B), the ones we consider here
+are included for being especially pioneering or popular (or both).  Here, we list those libraries,
+indicate which version we considered, and say a few words about why we included it in the analysis.
+
+- [**Boost Units**](https://www.boost.org/doc/libs/1_82_0/doc/html/boost_units.html) (version:
+  1.82.0)
+    - One of the longest-standing C++ unit libraries, and the most prominent pre-C++14 option.
+- [**nholthaus/units**](https://github.com/nholthaus/units) (version: 2.3.3)
+    - Kicked off the revolution in modern (that is, post-C++11 watershed) units libraries.
```

Comment:
What does the word "watershed" add here? It leads me to believe that C++11 was particularly revolutionary, but then the word is again used below to describe the C++20 era.

Reply:
That was my intention.  I consider both C++11 and C++20 to be revolutionary releases.

I'm fine keeping it, but am also open to wording suggestions, as I wasn't completely happy with this even before your comment.